README.MD

```diff
@@ -4,15 +4,105 @@ This plugin is only needed if you want to host a public SRV mode proxy instance.
 
 ## SRV ViaProxy
 ### Running ViaProxy in SRV mode
-To use the SRV mode in ViaProxy you have to start it through the command line.\
-The command line arguments are as follows:
-````bash
-java -jar ViaProxy.jar -ba 0.0.0.0 -bp <bind port> -srv -oam_auth -a0 -v1.8.x
-````
+> [!CAUTION]
+> This Option is meaningless because there is no proper authentication method for 1.19.4+ as OpenAuthMod discontinued and because ViaProxy was open source for 1 year is easier to use on client without their help. 
```

Comment:
I don't think this statement is true. There are still uses for SRV mode. I use it for quickly testing stuff without having to download and setup ViaProxy. While it is not directly public, I still want to minimize the risk of people discovering it and using it for accessing local services.

Reply:
oh wait a minute, i realized that My ViaProxy Instance uses SRV mode too.
whoops, my bad.

Reply:
fixed.
total 4 commits.